NET.A.Catalog/NET.A.Catalog/Controllers/ItemsController.cs

```diff
@@ -51,11 +58,11 @@ public async Task<ActionResult<int>> Create(CreateItemCommand command)
         }
 
         [HttpPut]
-        [Route("item/{id}", Name = "update-item")]
+        [Route("item", Name = "update-item")]
         public async Task<ActionResult> Update(UpdateItemCommand command)
         {
             await Mediator.Send(command);
-
+            _messageProducer.SendMessage(command);
```

Comment:
Probably mediator command handler is better place to send business specific events.

Reply:
I was thinking about it but feels like it's not the responsibility of the Mediator to send data outside the service
Maybe a proxy service that has a reference to the Mediator and to the Message Bus can be a solution
But I don't think I need to change the Mediator